src/viperleed/gui/measure/hardwarebase.py

```diff
@@ -247,6 +242,13 @@ def disconnected_slot(slot, signal, *more_signals, type=None):
             safe_connect(signal, slot, type=type)
 
 
+def get_default_path():
+    """Return the path to default settings."""
+    # The path is detected at runtime and not set as a global variable
+    # to catch any changes that may occur to the path in runtime.
+    return Path(get_qsettings('Measurement').fileName()).parent.resolve()
```

Comment:
```suggestion
    return Path(get_qsettings('Measurement').fileName()).resolve().parent
```

Better to `.resolve()` before `.parent`